1.EDA/SH_EDA_report2.ipynb

```
@@ -0,0 +1,106 @@
+{
+ "cells": [
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)